pkg/apis/networkaddonsoperator/shared/zz_generated.deepcopy.go

```diff
@@ -24,7 +24,7 @@ package shared
 import (
 	configv1 "github.com/openshift/api/config/v1"
 	conditionsv1 "github.com/openshift/custom-resource-status/conditions/v1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
needed?

Reply:
not really, this is a tailing fix, since CNAO does not check fmt checks on PRs.. 
If you insist I will remove and put in another PR

Reply:
can you try please to revert this file ? it doesn't seem related to this PR ?